src/api/parser/src/feed/processor.js

```diff
@@ -0,0 +1,214 @@
+/**
+ * A processor function to be run concurrently, in its own process, and
+ * with potentially multiple simultaneous instances, by the feed queue.
+ * https://github.com/OptimalBits/bull#separate-processes
+ */
+
+const { parse } = require('feedparser-promised');
+const fetch = require('node-fetch');
+
+const { logger } = require('@senecacdot/satellite');
```

Comment:
Move this up with the other 3rd party defines, and new line between them and local ones.